index.js

```diff
@@ -50,6 +50,10 @@ module.exports = (bookshelf, options = {}) => {
         destroyed: 'destroying'
       }
 
+      if (!Array.isArray(this.historyOptions.autoHistory)) {
```

Comment:
This fixes a bug where `false` and `null` where being passed in for `autoHistory`.